config.md

```diff
@@ -143,3 +143,27 @@ clang's parser, such as incoming references.
 Whether files are built in the background to produce a project index.
 This is checked for translation units only, not headers they include.
 Legal values are `Build` (the default) or `Skip`.
+
+### External
+
+Used to define an external index source. This can be:
```

Comment:
"This can be" could be rephrased somehow but I don't know what would be the best variant. `index source, such as:` or a better connection via `Index source can be:`. Not important, feel free to leave as is.

Reply:
completely dropped the second sentence and put `index source:`